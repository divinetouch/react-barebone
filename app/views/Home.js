import React, {Component} from 'react';

let Home = class extends Component {
    render() {
        return (
            <div>
                <h1>Welcome to Home page</h1>
            </div>
        );
    }
};

export default Home;