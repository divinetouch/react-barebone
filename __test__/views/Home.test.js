import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../app/views/Home.js';


it('should produce a welcom to home page div', () => {
    const component = renderer.create(<Home />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});