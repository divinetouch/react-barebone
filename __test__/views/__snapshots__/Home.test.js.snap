exports[`test should produce a welcom to home page div 1`] = `
<div>
  <h1>
    Welcome to Home page
  </h1>
</div>
`;
